crm_rma_prodlot_invoice/tests/test_crm_rma_prodlot_invoice.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2015 Vauxoo
+#    Author: Yanina Aular, Osval Reyes
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.tests.common import TransactionCase
+from datetime import date
+
+
+class TestCrmRmaProdLotInvoice(TransactionCase):
+
+    """
+    Test Cases
+    """
+
+    def setUp(self):
+        super(TestCrmRmaProdLotInvoice, self).setUp()
+        self.sale_order = self.env['sale.order']
+        self.invoice_line = self.env['account.invoice.line']
+        self.partner_id = self.env['res.partner'].browse(
+            self.ref('base.res_partner_2'))
+        self.voucher = self.env['account.voucher']
+        self.cash_type_journal_id = self.env['account.journal'].\
+            search([('type', '=', 'cash')], limit=1)[0]
+        self.product_id = self.env['product.product'].\
+            browse(self.ref('product.product_product_8'))
+
+        date_start = date.today().replace(day=1, month=1).strftime('%Y-%m-%d')
+        self.period_id = self.env['account.fiscalyear'].search(
+            [('date_start', '=', date_start)]).period_ids[8]
+
+        self.account_id = self.env['account.account'].\
+            browse(self.ref("account.a_recv"))
+        self.pay_account_id = self.env['account.account'].\
+            browse(self.ref("account.cash"))
+        self.journal_id = self.env['account.journal'].\
+            browse(self.ref("account.bank_journal"))
+
+    def test_01_action_ship_create(self):
+        """
+        Create sale order with Before Delivery policy
+        """
+        sale_order_id = self.sale_order.create({
+            'partner_id': self.partner_id.id,
+            'client_order_ref': 'TEST_SO',
+            'order_policy': 'prepaid',  # 'before delivery' invoice creation
```

Comment:
@yaniaular at this time, when replacing 'product_unique_serial' in place of 'account_anglo_saxon' i'm getting this error ```ValueError: Wrong value for sale.order.order_policy: 'prepaid'``` also I verified ```order_policy``` selection values a got the following
```bash
(Pdb) self.env['sale.order']._fields['order_policy'].selection
[('manual', 'On Demand')]
```
Modules ```sale_stock``` and ```crm_rma_prodlot_invoice``` are installed